integration/mediation-tests/tests-transport/src/test/java/org/wso2/carbon/esb/jms/inbound/transport/test/JMSInboundBrokerShutdownTestCase.java

```diff
@@ -46,9 +46,11 @@ protected void init() throws Exception {
                 new ServerConfigurationManager(new AutomationContext("ESB", TestUserMode.SUPER_TENANT_ADMIN));
     }
 
+    // Disabling the test-case due to no proper fix
```

Comment:
Please add a descriptive comment why test is dissabled

Reply:
Fixed 86254d8a95ae63590c17f4efe4047baac6b441c6